lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -360,3 +360,56 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['media', '/^my-/', /^custom-/i] }],
```

Comment:
Let's use one string and one regex:

```suggestion
	config: [true, { ignoreAtRules: ['supports', /^my-/] }],
```